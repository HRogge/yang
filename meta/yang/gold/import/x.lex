module "module"
[ident] "x"
{ "{"
prefix "prefix"
[string] "\"\""
; ";"
namespace "namespace"
[string] "\"\""
; ";"
import "import"
[ident] "one"
; ";"
revision "revision"
[revision] "0"
; ";"
import "import"
[ident] "two"
{ "{"
prefix "prefix"
[string] "\"q\""
; ";"
revision "revision"
[revision] "0"
; ";"
description "descriptio"...
[string] "\"d\""
; ";"
reference "reference"
[string] "\"99\""
; ";"
} "}"
leaf "leaf"
[ident] "x"
{ "{"
type "type"
[ident] "string"
; ";"
} "}"
} "}"
