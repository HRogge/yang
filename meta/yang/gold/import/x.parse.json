{
"module":{
  "ident":"x",
  "revision":{
    "rev-date":"0"},
  "definitions":[
    {
      "ident":"x",
      "leaf":{
        "ident":"x",
        "type":{
          "ident":"string",
          "format":"string"}}},
    {
      "ident":"one:g",
      "uses":{
        "ident":"one:g"}},
    {
      "ident":"two:g",
      "uses":{
        "ident":"two:g"}}]}}