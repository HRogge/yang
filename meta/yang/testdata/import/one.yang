module one {
    prefix "";
    namespace "";
    revision 0;

    grouping g {
        leaf y {
            type string;
        }
    }
}