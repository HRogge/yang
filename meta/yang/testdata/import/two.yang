module two {
    prefix "";
    namespace "";
    revision 0;

    leaf z {
        type string;
    }
}