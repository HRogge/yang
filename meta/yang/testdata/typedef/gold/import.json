{
"module":{
  "ident":"import",
  "revision":{
    "rev-date":"0"},
  "dataDef":[
    {
      "ident":"l",
      "leaf":{
        "type":{
          "ident":"two:t",
          "format":"string"}}}]}}