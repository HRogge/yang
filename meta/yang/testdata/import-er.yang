module import-er {
    prefix "";
    namespace "";
    import import-ee1;
    revision 0;
    import import-ee2;

    leaf x {
        type string;
    }
}