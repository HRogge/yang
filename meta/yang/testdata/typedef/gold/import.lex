module "module"
[ident] "import"
{ "{"
revision "revision"
[string] "0"
; ";"
import "import"
[ident] "import-1"
{ "{"
prefix "prefix"
[string] "one"
; ";"
} "}"
uses "uses"
[ident] "one:g"
; ";"
} "}"
