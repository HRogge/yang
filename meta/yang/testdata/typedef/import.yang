module import {
    revision 0;
    import import-1 {
        prefix one;
    }

    uses one:g;
}