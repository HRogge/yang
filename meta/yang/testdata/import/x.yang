module x {
    prefix "";
    namespace "";
    import one;
    revision 0;
    import two {
        prefix "q";
        revision 0;
        description "d";
        reference "99";
    }

    leaf x {
        type string;
    }
}