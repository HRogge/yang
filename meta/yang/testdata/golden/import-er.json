{
"module":{
  "ident":"import-er",
  "revision":{
    "rev-date":"0"},
  "definitions":[
    {
      "ident":"y",
      "leaf":{
        "ident":"y",
        "type":{
          "ident":"string"}}},
    {
      "ident":"z",
      "leaf":{
        "ident":"z",
        "type":{
          "ident":"string"}}},
    {
      "ident":"x",
      "leaf":{
        "ident":"x",
        "type":{
          "ident":"string"}}}]}}